src/plugins/telemetryTable/components/table-column-header.vue

```diff
@@ -0,0 +1,179 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+<template>
+<th 
+    :style="{ width: columnWidth + 'px', 'max-width': columnWidth + 'px'}"
+    :draggable="isEditing"
+    @mouseup="sort"
+    v-on="isEditing ? {
+        dragstart: columnMoveStart,
+        drop: columnMoveEnd,
+        dragleave: hideDropTarget,
+        dragover: ($event) => dragOverColumn($event.currentTarget, $event)
```

Comment:
@akhenry is there a reason to extract $event.currentTarget here instead of in the dragOverColumn method?

Reply:
Good catch. This was necessary when the function was throttled. [currentTarget is a transient property and it was getting lost](https://github.com/jashkenas/underscore/issues/1905). I don't believe it's necessary now.